### ISSUE TITLE
Missing logo for SuperVPNGeckoVPN data aggregator breach

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blurts-server#2059]
Ref: https://monitor.firefox.com/breach-details/SuperVPNGeckoVPN

```sh
npx pdehaan/blurts-missing-logos -c

API: https://monitor.firefox.com/hibp/breaches
CDN: https://monitor.cdn.mozilla.net

[
  {
    "logo": "https://monitor.cdn.mozilla.net/img/logos/Gab.png",
    "details": "https://monitor.cdn.mozilla.net/breach-details/Gab",
    "status": 404
  },
  {
    "logo": "https://monitor.cdn.mozilla.net/img/logos/MeetMindful.png",
    "details": "https://monitor.cdn.mozilla.net/breach-details/MeetMindful",
    "status": 404
  },
  {
    "logo": "https://monitor.cdn.mozilla.net/img/logos/SuperVPNGeckoVPN.png",
    "details": "https://monitor.cdn.mozilla.net/breach-details/SuperVPNGeckoVPN",
    "status": 404
  }
]
```

```sh
npx pdehaan/blurts-server-data-classes

Missing "device-serial-numbers" data class from SuperVPNGeckoVPN breach
Missing "login-histories" data class from SuperVPNGeckoVPN breach
```

_Originally posted by @pdehaan in https://github.com/mozilla/blurts-server/issues/2058#issuecomment-790906144_

**Comment by pdehaan** [comment]
Missing strings added in https://github.com/mozilla/blurts-server/pull/2061

**Comment by pdehaan** [comment]
Fixed via #2060 

![Firefox_Monitor](https://user-images.githubusercontent.com/557895/110033883-d145b800-7cee-11eb-8636-51242a202141.png)
